src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/0.1.0/Generated.PowerShell.Commands/SwaggerPathCommands/Get-EdgeGateway.ps1

```diff
@@ -0,0 +1,231 @@
+<#
+The MIT License (MIT)
+
+Copyright (c) 2017 Microsoft
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+#>
+
+Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath .. | Join-Path -ChildPath .. | Join-Path -ChildPath "GeneratedHelpers.psm1")
+<#
+.DESCRIPTION
+    Get edge gateways.
+
+.PARAMETER Filter
+    OData filter parameter.
+
+.PARAMETER EdgeGateway
+    name of the edge gateway.
+
+.PARAMETER Skip
+    Skip the first N items as specified by the parameter value.
+
+.PARAMETER Location
+    Location of the resource.
+
+.PARAMETER Top
+    Return the top N items as specified by the parameter value. Applies after the -Skip parameter.
+
+.EXAMPLE
+
+Get-AzsEdgeGateway -Location "local"
```

Comment:
So, I'm not super excited abotu this.  We want the md files to be the source of this kind of text, this gives us two sources.

Reply:
All comments (other than examples) are auto-generated.  I talked to the PSSwagger team and they said they will not change this and leave it up to the client to generate anything extra.